.github/workflows/codespell.yml

```diff
@@ -0,0 +1,9 @@
+name: codespell
```

Comment:
I like to put a line or two of comment on an action, for those who won't know what `codespell` is doing.

Reply:
```suggestion
# https://pypi.org/project/codespell will fix common misspellings in text files. It’s designed primarily
# for checking misspelled words in source code, but it can be used with other files as well.
name: codespell
```